pytest_fixtures/component/provision_pxe.py

```diff
@@ -105,7 +118,7 @@ def module_provisioning_rhel_content(
     rhel_xy = Version(
         constants.REPOS['kickstart'][f'rhel{rhel_ver}']['version']
         if rhel_ver == 7
-        else constants.REPOS['kickstart'][f'rhel{rhel_ver}_bos_beta']['version']
+        else '10.1'  # EL10 pre-release version
```

Comment:
Instead of hardcoding `10.1`, keep it as constant and use the constant here. Good coding practice.

Reply:
Good point, moved this to constants, thanks!